### ISSUE TITLE
Enhance EloRatingCalculator

### ISSUE THREAD

**Issue by ZsoltToth** [Coeus-Consulting/spring-tutorial-elo-rating-system-corvinus#6]
Enhance the EloRatingCalculator implementation to allow its configuration from DB. 

Blocked by: #5 

Note: Not required. Nice to have feature.

AC:
 - Adjustment rate is read from a Config Table
